actions/src/main/kotlin/com/alfresco/content/actions/ContextualActionsSheet.kt

```diff
@@ -33,6 +33,7 @@ class ContextualActionsSheet : BottomSheetDialogFragment(), MavericksView {
         return binding.root
     }
 
+    @Deprecated("Deprecated in Java")
```

Comment:
Please create a separate ticket to upgrade the Kotlin version to resolve these issues

Reply:
I have created this ticket for upgradation
https://hyland.atlassian.net/browse/ADST-598